app/src/main/java/com/google/samples/apps/sunflower/adapters/PlantDetailBindingAdapter.kt

```diff
@@ -37,9 +37,12 @@ fun imageFromUrl(view: ImageView, imageUrl: String?) {
     }
 }
 
-@BindingAdapter("goneIf")
-fun goneIf(view: FloatingActionButton, isGone: Boolean?) {
-    if (isGone == null || isGone) view.hide() else view.show()
+@BindingAdapter("minusIf")
+fun minusIf(view: FloatingActionButton, isPlanted: Boolean?) {
+    if (isPlanted == null || !isPlanted)
```

Comment:
i prefer to put the braces in the if-else statement if more than 1 line.
this is also mentioned in https://android.github.io/kotlin-guides/style.html#formatting